packages/drivers/routerlicious-driver/src/documentStorageService.ts

```diff
@@ -87,12 +87,16 @@ export class DocumentStorageService implements IDocumentStorageService {
         throw new Error("NOT IMPLEMENTED!");
     }
 
-    public async createBlob(file: Uint8Array): Promise<ICreateBlobResponse> {
-        const response = this.manager.createBlob(Uint8ArrayToString(file, "base64"), "base64");
+    public async createBlob(file: ArrayBufferLike): Promise<ICreateBlobResponse> {
+        const response = this.manager.createBlob(
+            Uint8ArrayToString(
+                new Uint8Array(file), "base64"),
```

Comment:
(Note that 'new Uint8Array()' does not copy the underlying data...  although transcoding to base64 certainly does.)